### PR TITLE
bugfix s56_limit_ch4_n2o_price default value

### PR DESCRIPTION
**Pull request by flohump**
## :bird: Description of this PR :bird:

- changed default value for s56_limit_ch4_n2o_price from 1000 to 4000 for consistency with c57_macc_version = "PBL_2022"
- FSDP runs are feasible

## :wrench: Checklist for PR creator :wrench:

- [x] Label pull request [from the label list](https://github.com/magpiemodel/magpie/labels).
  - **Low risk**: Simple bugfixes (missing files, updated documentation, typos) or changes  in start or output scripts
  - **Medium risk**: Uncritical changes in the model core (e.g. moderate modifications in non-default realizations)
  - **High risk**: Critical changes in model core or default settings (e.g. changing a model default or adjusting a core mechanic in the model)

Default run is unchanged.

Productivity_Landuse_Intensity_Indicator_Tau-67
Resources_Land_Cover_Cropland-77

As expected, there is slightly more non-CO2 emission reduction in climate policy runs the updated version.
 
Emissions_CH4_Land_Agriculture-4
Emissions_N2O_Land_Agriculture-2

- [x] Self-review own code
  - No hard coded numbers and cluster/country/region names.
  - The new code doesn't contain declared but unused parameters or variables.
  - [`magpie4`](https://github.com/pik-piam/magpie4) R library has been updated accordingly and backwards compatible where necessary.
  - `scenario_config.csv` has been updated accordingly (important if `default.cfg` has been updated)

- [x] Document changes 
  - Add changes to `CHANGELOG.md`
  - Where relevant, put In-code documentation comments
  - Properly address updates in interfaces in the module documentations
  - run [`goxygen::goxygen()`](https://github.com/pik-piam/goxygen) and verify the modified code is properly documented

- [x] Perform test runs
  - **Low risk**: 
    - Run a compilation check via `gams main.gms action=c`
  - **Medium risk**: 
    - Run test runs via `Rscript start.R --> "test runs"`
  - **High risk**:
    - Run test runs via `Rscript start.R --> "test runs"`
    - Default run based on the current version of the fork from which PR is made
    - Provide relevant comparison plots (land-use, emissions, food prices, land-use intensity,...)

### :chart_with_downwards_trend: Performance changes :chart_with_upwards_trend:
  
  - Current develop branch default : 0.39 h
  - This PR's default :  0.42 h

newplot-12

## :rotating_light: Checklist for reviewer :rotating_light:

- PR is labeled correctly
- Code changes look reasonable
  - No hard coded numbers and cluster/country/region names.
  - No unnecessary increase in module interfaces
  - model behavior/performance is satisfactory.
- Changes are properly documented
  - `CHANGELOG` is updated correctly
  - Updates in interfaces have been properly addressed in the module documentations
  - In-code documentation looks appropriate

### CODE REVIEW

**Review by pfuehrlich-pik — APPROVED** [review]

**Review by FelicitasBeier — APPROVED** [review]
Looks good

File: CHANGELOG.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### fixed
 - **scripts** rewrite of land conversion cost calibration script `landconversion_cost.R`
-
+- **config** changed default value for s56_limit_ch4_n2o_price from 1000 to 4000 for consistency with c57_macc_version = "PBL_2022"
```
Comment (bodirsky) [inline CHANGELOG.md:23]: unit missing. is it the same for n2o as for ch4?

**Review by bodirsky — COMMENTED** [review]
please add unit to config cfg

File: CHANGELOG.md (hunk 1 above)
Comment (flohump) [reply CHANGELOG.md:23]: The unit is in the default.cfg
The limit is defined in USD05MER per tC equivalent, which is translated into corresponding limits for CH4 and N2O with GWP100 conversion factors in the code.

**Pull request merged**